### PR TITLE
[ACR] `az acr connected-registry install info/renew-credentials`: Add new parameter `--parent-protocol` to specify parent protocol

### PR DESCRIPTION
**Pull request by xyxyxyxyxyxy**
**Description**
Add specifying parent protocol to 'acr connected-registry install' to fetch complete connection string.
Update the help instruction and argument description
Fix some typos.

**Testing Guide**
az acr connected-registry install info -n -r --parent-protocol
az acr connected-registry install renew-credentials -n -r --parent-protocol

**History Notes**

[ACR] BREAKING CHANGE: az acr connected-registry install info: Add a new required parameter --parent-protocol.
[ACR] BREAKING CHANGE: az acr connected-registry install renew-credentials: Add a new required parameter --parent-protocol.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ACR

File: src/azure-cli/azure/cli/command_modules/acr/_params.py [hunk 1]
```diff
@@ -436,6 +436,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('remove_repos', options_list=['--remove'], nargs='*', required=False,
                    help='respsitory permissions to be removed from the targeted connected registry and it\'s succesors sync scope maps. Use the format "--remove [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
 
+    with self.argument_context('acr connected-registry install') as c:
+        c.argument('parent_protocol', options_list=['--parent-protocol'], help='Required parameter to specify the parent protocol. Should be either "http" or "https".', required=True)
```
Comment (northtyphoon) [inline src/azure-cli/azure/cli/command_modules/acr/_params.py:440]: Can the argument be defined as an enum type? Something like https://github.com/Azure/azure-cli/blob/29767d75d850ddc1c24cc85bd46d861b61d77a47/doc/authoring_command_modules/authoring_commands.md#registering-enums

Comment (northtyphoon) [reply src/azure-cli/azure/cli/command_modules/acr/_params.py:440]: @rosanch @huanwu @jaysterp

Comment (xyxyxyxyxyxy) [reply src/azure-cli/azure/cli/command_modules/acr/_params.py]: Changed

File: src/azure-cli/azure/cli/command_modules/acr/_help.py [hunk 2]
```diff
@@ -1317,18 +1317,18 @@
 type: command
 short-summary: Retrieves information required to activate a connected registry.
 examples:
-  - name: Prints the values requiered to activate a connected registry in json format
+  - name: Sets http as the parent protocol, and prints the values required to activate a connected registry in json format
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/acr/_help.py:1320]: Please the first person voice
```suggestion
  - name: Set http as the parent protocol, and prints the values required to activate a connected registry in json format
```

File: src/azure-cli/azure/cli/command_modules/acr/_help.py [hunk 3]
```diff
@@ -1317,18 +1317,18 @@
...
-        az acr connected-registry install info --registry mycloudregistry --name myconnectedregistry
+        az acr connected-registry install info --registry mycloudregistry --name myconnectedregistry --parent-protocol http
 """
 
 helps['acr connected-registry install renew-credentials'] = """
 type: command
 short-summary: Retrieves information required to activate a connected registry, and renews the sync token credentials.
 examples:
-  - name: Prints the values in json format requiered to activate a connected registry and the newly generated sync token credentials.
+  - name: Sets http as the parent protocol, and prints the values in json format required to activate a connected registry and the newly generated sync token credentials.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/acr/_help.py:1329]:
```suggestion
  - name: Set http as the parent protocol, and prints the values in json format required to activate a connected registry and the newly generated sync token credentials.
```

**Comment by zhoxing-ms** [comment]
@xyxyxyxyxyxy Could you add some new test cases for this new parameter?

**Review by zhoxing-ms — APPROVED** [review]
I can merge this PR first. Please add relevant tests later

**Comment by xyxyxyxyxyxy** [comment]
Thanks, we will add tests later

**Pull request merged**